cmd/earthly/main.go

```diff
@@ -704,6 +705,12 @@ func newEarthlyApp(ctx context.Context, console conslogging.ConsoleLogger) *eart
 				},
 			},
 		},
+		{
+			Name:      "ls",
+			Usage:     "List targets from an earthfile *experimental*",
+			UsageText: "earthly [options] ls [<import-path>]",
```

Comment:
Does "import-path" here mean the path can also be the path to the Earthfile, like `earthly ls ./subdir/Earthfile`? Or is it strictly path to the directory where the Earthfile is?

Wondering if the latter should even work -- I can see it being useful but maybe not worth the extra complexity here?

Reply:
I took a closer look at our docs, and under https://docs.earthly.dev/docs/guides/target-ref we call this `<project-ref>`. I'll rename import-path to project-ref.

I wanted to keep it consistent with the rest of our target referencing (where we never type out the word "Earthfile" on our targets). We could potentially trim any `/Earthfile` suffix if it's passed in by our users. But then that causes us to diverge from what a project-reference is.

Reply:
thinking it over a bit more, it's a pretty easy thing to do, let's see if we get a follow up bug report for "unable to list targets from `Earthfile` contained in sub-directory named `Earthfile` (i.e. `earthly ls ./Earthfile/Earthfile`).